src/frontend/src/env/tokens.icrc.json

```diff
@@ -1,4 +1,9 @@
 {
-	"production": {},
+	"production": {
+		"BURN": {
+			"indexCanisterId": "nrant-tyaaa-aaaag-atsjq-cai",
+			"ledgerCanisterId": "egjwt-lqaaa-aaaak-qi2aa-cai"
```

Comment:
Similar to SNS tokens, is there any plan to pre-render the metadata of the ledgers we statically add to OISY to avoid a query and an update call for loading the metadata at runtime?

Reply:
I assumed we were already pre-fetching metadata for ICRC but, we actually only do so for SNS. So this might be a nice improvement afterwards but, not strictly related to this task.